Lecture2/src/Task3/Task3App.java

```diff
@@ -0,0 +1,75 @@
+package Task3;
+
+import java.util.Scanner;
+
+public class Task3App {
+
+    private static final char SYMBOL = '*';
+    private static final char SPACE = ' ';
+    private static final int ONE = 1;
+    private static final int TWO = 2;
+    private static int buff1;
+    private static int buff2;
+    private static int sideSize;
+    private static int maxLength;
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+
+        do {
+            System.out.print("Enter side size: ");
+            while (!scanner.hasNextInt()) {
+                scanner.next();
+                System.out.println("You entered not a number!");
+            }
+            sideSize = scanner.nextInt();
+            if (sideSize > 1) {
+                break;
+            } else {
+                System.out.println("Side size must be more than 2!");
+            }
+        } while (true);
+
+        initVar();
```

Comment:
not clear what `initVar ` method  responsible for

Reply:
Deleted this method. Variables initialize in the main method now.